### DISCUSSION TITLE
"Administrator" password in the Gnome GUI

### DISCUSSION THREAD

**Discussion by litebito** [Security-Onion-Solutions/securityonion#9955] [Unsupported Versions]
Hi,

I just finished setting up SO on a vm as a standalone installation, and I did a so-analyst-install after that.
So far, everything seems to work ok, except for 1 thing, I was trying to fix some settings for the GUI, like the keyboard for the login screen. 
However, it seems to need another password for a user name "Administrator" 
![image](https://user-images.githubusercontent.com/3867999/226131273-65d1a9ea-e48a-4363-87cf-44ed33098332.png)
I did try the passwords I specified during the setup of SO, but none of them are accepted for this user.

Is there a default password for this user?

(I did search the issues and discussions with this question, but didn't find any answers so far)

**Comment by isaacgolding** [comment]
That would be your admin password for your HOST system.

**Comment by litebito** [comment]
Hi,
I'm not sure what you mean here. 
the HOST system is the ESX in my case
I created a bare metal VM
I setup SO on a "bare metal" VM using the Installation ISO
I tried all passwords which I was prompted to create during the the installation of SO

**Reply by isaacgolding** [reply]
Security Onion doesn't come with a Gnome interface.   If you installed the official Security Onion ISO it only comes with a shell/ssh interface and a web interface.

**Comment by litebito** [comment]
Yes it does, as noted in my first post, I issued the command **"so-analyst-install**" which as the SO [documentation ](https://docs.securityonion.net/en/2.3/analyst-vm.html)explains, adds a gnome based desktop for SO.
(it is not installed during the standard SO installation, but can be installed as an option)
From the SO documentation : 
The so-analyst-install script will install a full GNOME desktop environment including Chromium web browser, [NetworkMiner](https://docs.securityonion.net/en/2.3/networkminer.html#networkminer), [Wireshark](https://docs.securityonion.net/en/2.3/wireshark.html#wireshark), and other analyst tools. so-analyst-install is totally independent of the standard setup process, so you can run it before or after setup or not run setup at all if all you really want is the Analyst VM itself.

**Reply by isaacgolding** [reply]
My mistake.   You are correct.  So this password entry "should" be your login password for your user account.  If that doesn't work then I'm afraid I don't have any easy answers for you.

**Reply by litebito** [reply]
I was afraid of that.... I tried all of the passwords which I was prompted during any install part.
I tried wiping and reinstalling everything from bare metal 3 times ..... so I guess a bug somewhere or we are not allowed admin access in gnome.
I'll try to raise it as an issue

**Comment by InfosecGoon** [comment]
Is there actually a user named Administrator on the system? You can check in /etc/passwd.
